### PR TITLE
chore: remove slack connect and call help

### PR DESCRIPTION
**Pull request by makeavish**
### Summary

Remove slack connect and call help

### CODE REVIEW

File: frontend/src/pages/Support/Support.tsx [hunk 1]
```diff
@@ -10,8 +10,8 @@ import useLicense from 'hooks/useLicense';
 import { useNotifications } from 'hooks/useNotifications';
 import {
 	Book,
-	Cable,
-	Calendar,
+	// Cable,
```
Comment (vikrantgupta25) [inline frontend/src/pages/Support/Support.tsx:13]: remove commented code

Comment (makeavish) [reply frontend/src/pages/Support/Support.tsx]: Removed

**Review by ahmadshaheer — APPROVED** [review]

**Comment by YounixM** [comment]
@vishalsharma1800 : Please add before and after screen recording for the change.

**Comment by YounixM** [comment]
Have we checked this for all types of accounts?

**Comment by makeavish** [comment]
@YounixM Here's loom video: https://www.loom.com/share/d61275ef02184acc946388aa07543b3b?sid=a56e408d-b318-4d0b-b079-de6bbfe3e9a7

**Comment by makeavish** [comment]
> Have we checked this for all types of accounts?

This PR doesn't touch any user access related things

**Review by YounixM — APPROVED** [review]

**Pull request merged**